core/src/main/java/com/google/errorprone/bugpatterns/IdentifierName.java

```diff
@@ -278,6 +278,9 @@ private static boolean isConformant(Symbol symbol, String name) {
     if (isStaticVariable(symbol) && isConformantStaticVariableName(name)) {
       return true;
     }
+    if (name.isEmpty()) {
```

Comment:
`isConformant` is also called for methods. That seems ok; `javac` does not accept an absent method name and rejects the meta-name `_` with

```
/Test.java:5: error: underscore not allowed here
  void _() {
```

and if, improbably, that ever did change, presumably an unnamed method should be treated no differently from an unnamed variable.

A more difficult question is whether something without a name can "conform" to a naming scheme. It seems more like asking the question in the first place is nonsensical. So perhaps this check should be at a higher level inside the checker, and perhaps it should be more explicitly `isUnnamed(name)`.

Reply:
> A more difficult question is whether something without a name can "conform" to a naming scheme.

I think most definitions you'd give of, say, lowerCamelCase would pass for the empty string, so I think the answer here is just "yes".